.changesets/docs_bnjjj_docs_standard_instr.md

```diff
@@ -0,0 +1,22 @@
+### Standard instrument configuration documentation for subgraphs ([PR #5422](https://github.com/apollographql/router/pull/5422))
+
+Added documentations about standard instruments available at the subgraph service:
```

Comment:
```suggestion
Added documentation about standard instruments available at the subgraph service level:
```

Reply:
(I believe we've historically used "at" for talking about a "service level" ([e.g.](https://github.com/apollographql/router/blob/dev/CHANGELOG.md#-documentation-8)) and "in" when talking about something happening during or in a service. ([e.g.](https://www.apollographql.com/docs/router/configuration/telemetry/instrumentation/instruments/#value)))